source/retryable-reads/tests/README.rst

```diff
@@ -81,17 +81,21 @@ Each YAML file has the following keys:
     the default is all topologies (i.e. ``["single", "replicaset", "sharded",
     "load-balanced"]``).
 
-  - ``serverless``: Optional string. Whether or not the test should be run on
-    serverless instances imitating sharded clusters. Valid values are "require",
-    "forbid", and "allow". If "require", the test MUST only be run on serverless
-    instances. If "forbid", the test MUST NOT be run on serverless instances. If
-    omitted or "allow", this option has no effect.
-
-    The test runner MUST be informed whether or not serverless is being used in
-    order to determine if this requirement is met (e.g. through an environment
-    variable or configuration option). Since the serverless proxy imitates a
-    mongos, the runner is not capable of determining this by issuing a server
-    command such as ``buildInfo`` or ``hello``.
+  - ``serverless``: (optional): Whether or not the test should be run on Atlas
+    Serverless instances. Valid values are "require", "forbid", and "allow". If
+    "require", the test MUST only be run on Atlas Serverless instances. If
+    "forbid", the test MUST NOT be run on Atlas Serverless instances. If omitted
+    or "allow", this option has no effect.
+
+    The test runner MUST be informed whether or not Atlas Serverless is being
+    used in order to determine if this requirement is met (e.g. through an
+    environment variable or configuration option).
+
+    Note: the Atlas Serverless proxy imitates mongos, so the test runner is not
+    capable of determining if Atlas Serverless is in use by issuing commands
+    such as ``buildInfo`` or ``hello``. Furthermore, connections to Atlas
+    Serverless use a load balancer, so the topology will appear as
+    "load-balanced".
```

Comment:
FYI @isabelatkinson: @kevinAlbs requested I copy/paste the new text from the unified test format spec to these older test READMEs. There's probably not much to review here if you'd rather skip.